src/main/java/de/tum/in/www1/artemis/repository/CourseRepository.java

```diff
@@ -52,6 +52,15 @@ public interface CourseRepository extends JpaRepository<Course, Long> {
     @Query("select distinct course from Course course where course.studentGroupName like :name")
     Course findCourseByStudentGroupName(@Param("name") String name);
 
+    @Query("select distinct course from Course course where course.instructorGroupName = :name")
```

Comment:
We usually use text blocks """ """ for SQL queries and capital letters for SQL keywords. Would be good if you change it for consistency, ideally for both your new queries and the old ones you encounter.

Reply:
Changed to """"" text blocks